cmake/generic.cmake

```diff
@@ -88,7 +88,7 @@
 #
 
 # including binary directory for generated headers.
-include_directories(${CMAKE_BINARY_DIR})
+include_directories(${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
generic.cmake  is included by paddle/CMakeLists.txt, so CMAKE_CURRENT_BINARY_DIR is same as PROJ_BINARY_ROOT